components/modules/Kconfig

```
@@ -14,6 +14,14 @@ menu "NodeMCU modules"
             Includes the bit module. This module provide bit manipulation
             functions on Lua numbers.
 
+    config NODEMCU_CMODULE_BLE
+        bool "BlueTooth GAP/GATT interface module"
```

```suggestion
        bool "Bluetooth GAP/GATT interface module"
```
The official term sees this as a single word.